### DISCUSSION TITLE
pip install not working?

### DISCUSSION THREAD

**Discussion by yobleck** [coqui-ai/TTS#1579] [General Q&A] [answered]
pip install -U TTS returns:
```
ERROR: Could not find a version that satisfies the requirement TTS
ERROR: No matching distribution found for TTS
```

python 3.10.4
pip 21.0
manjaro linux 5.15

Tried with and without -U and sudo.

**Comment by p0p4k** [comment, marked as answer]
```
git clone https://github.com/coqui-ai/TTS
pip install -e .[all,dev,notebooks]  # Select the relevant extras
```

Tried that?

**Reply by p0p4k** [reply]
Also, make virtual environment before installing anything. Report back your errors, if any. Thanks.

**Reply by yobleck** [reply]
Thanks, that worked. Do you happen to know why the readme says there is an option to just download from pip?

**Reply by lexkoro** [reply]
Because the pip package has currently following requirement  `Requires: Python >=3.6.0, <3.10`, but you have 3.10 installed, so it won't find a version that meets your Python version requirement.
